trp_project_scrum/views/menu_definitions.xml

```diff
@@ -9,5 +9,24 @@
             sequence="50" 
             groups="trp_project_scrum.collaborators_followers"
             />
+
+        <record id="action_show_projects" model="ir.actions.act_window">
+            <field name="name"> My projects - direct_access</field>
+            <field name="res_model">project.project</field>
+            <field name="view_type">form</field>
+            <field name="view_mode">tree,form</field>
+            <field name="view_id" eval="False"/>
+            <field name="domain" eval=""/>
+            <field name="context">{}</field>
+        </record>
```

Comment:
why don't you recycle https://github.com/OCA/OCB/blob/8.0/addons/project/project_view.xml#L314 (the only reason to create new actions is wanting different groups accessibility or different domains) or just add the collaborator group to https://github.com/OCA/OCB/blob/8.0/addons/project/project_view.xml#L8 ?